src/components/Editable.tsx

```diff
@@ -443,6 +444,14 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
     }
   }
 
+  /**
+   * Prevents onKeyDownAction call for shift, alt or ctrl keys.
+   */
+  const onKeyDown = (e: React.KeyboardEvent) => {
+    if (e.shiftKey || e.altKey || e.ctrlKey) return
```

Comment:
This incorrectly returns when pressing a modifier key in combination with another key, e.g. `Ctrl + RightArrow`. We only want to cancel when *only* a modifier key is pressed.

Also, you will need to add `e.metaKey` for Mac.